hooks/install.sh

```diff
@@ -2,4 +2,5 @@
 
 HOOKSDIR=./.git/hooks/
 
-cp ./hooks/pre-commit "$HOOKSDIR/pre-commit"
\ No newline at end of file
+rm -f "$HOOKSDIR/pre-commit"
+ln ./hooks/pre-commit "$HOOKSDIR/pre-commit"
```

Comment:
not `ln -s`? Or even `ln -f -s` and dropping the `rm`?

Reply:
Ah, that would probably be better.